src/svc_dg.c

```diff
@@ -279,6 +270,22 @@ svc_dg_rendezvous(SVCXPRT *xprt)
 		return (XPRT_DIED);
 	}
 
+        if (sp->sa_family == (sa_family_t) 0xffff) {
+                __warnx(TIRPC_DEBUG_FLAG_ERROR,
+                        "%s: Bad message sa_family is 0xffff",
+                        __func__);
+                svc_dg_xprt_free(su);
```

Comment:
This line should be removed, right ? We don't want to free UDP xprt when the issue happens, as we want UDP xprt to be available after issue happens. @dang what do you think ?

Reply:
I agree.  The point is to keep the xprt available for use after the bad message, so we should not be freeing the xprt.  This means XPRT_DIED is probably not the correct return, either (although the caller ignores the return value...)
This should probably return SVC_STAT(xprt);

Reply:
Thanks @madhuthorat @dang updated as requested.